### ISSUE TITLE
Fix increased flakiness in `Chrome_star_labs_applab_eyes3_eyes_Data Browser`

### ISSUE THREAD

**Issue by snickell** [code-dot-org/code-dot-org#59834]
Example [sauce labs failure](https://app.saucelabs.com/tests/437e73a69cce459fbc4cd3cb845bb130#57)

I believe this was introduced by #59667 , fix is _probably_ to increase timing a little bit in the data browser UI test.

Flakiness rate seems to be about 10%, so not terrible.

See for more detail: https://github.com/code-dot-org/code-dot-org/pull/59824#issuecomment-2232675855

**Comment by snickell** [comment]
Here's a fail: https://app.saucelabs.com/tests/437e73a69cce459fbc4cd3cb845bb130#63
Here's a pass: https://app.saucelabs.com/tests/0266d94f2fb04f218450a506b9b08d57#46

Difference is fail hangs on `I wait until element "th .test-tableNameDiv:contains(column1)" is visible` ([link to line in eyes3.feature](https://github.com/code-dot-org/code-dot-org/blob/9fc148484e19032e76c3b319ba8610ec2be2d768/dashboard/test/ui/features/star_labs/applab/eyes3.feature/#L20-L21)) forever:
![image](https://github.com/user-attachments/assets/39825078-016f-459e-8dde-d69b1bd09639)

**Comment by snickell** [comment]
Clicking manually through the UI, when I create a new table, I find that `$("th .test-tableNameDiv:contains(column1)").length` always returns 1, which means that it was found. I haven't been able to repro, maybe the quirk is in the definition of "is visible"?

I retried 25 times, but did not get a manual repro using jQuery.

To not make the header change state when clicking into the chrome devtools I had to enable "Emulate a focused page":
![image](https://github.com/user-attachments/assets/194aeeed-2844-4d6a-a7cc-448c04c461b6)

**Comment by snickell** [comment]
Ohhhhhh, maybe the issue is that the "enter key" press is supposed to remove the "input-like" state from the new column:
```
  When I press keys "foo" for element "#data-browser input"
  And I click selector "#data-browser button:contains(Add)"
  And I wait until element "#dataTable" is visible
  Then I see no difference for "data table"
```
<img width="611" alt="image" src="https://github.com/user-attachments/assets/caf94f14-464b-42bb-9f4c-05087c2ea39c">

```
  When I press enter key
  And I wait until element "th .test-tableNameDiv:contains(column1)" is visible
```

<img width="619" alt="image" src="https://github.com/user-attachments/assets/bab189c5-c8f4-43b6-82cb-a219a4f32a17">

Maybe the issue is that the enter key press (which accepts the column name?) is happening too quickly???

**Comment by snickell** [comment]
Based on this idea, I have a speculative fix in #59835, which shouldn't do any harm, and just waits 1s before pressing the enter button
